client/src/components/header/components/LoginLi/Login.js

```diff
@@ -0,0 +1,16 @@
+const Login = (props) => {
+    return ( 
+        <li
+            onClick={() => {
+                if (props.props.visibleLogin){
+                    props.props.setVisibleRegister(true)
+                    props.props.setVisibleLogin(false)
+                } else {
+                    props.props.setVisibleLogin(true)
+                }
+                
+            }}
+        >Login</li> );
```

Comment:
Overcomplicated logic, get rid of it. use Modals instead, search for a good package that provides you modals.